gotchas/styled-jsx-and-jest.md

```diff
@@ -0,0 +1,63 @@
+# TL;DR (recommended setup)
+
+### Usage of `styled-jsx` `css` and `css.resolve` function
+
+In your code-base, make sure to call the `resolve` function directly, like so:
```

Comment:
Should be 
> make sure to import the `resolve` function from the named export

Otherwise it could be interpreted as
```js
import css from 'styled-jsx/css'
const { resolve } = css
```